ext/openssl/tests/openssl_random_pseudo_bytes_basic.phpt

```diff
@@ -4,13 +4,11 @@ openssl_random_pseudo_bytes() tests
 <?php if (!extension_loaded("openssl")) print "skip"; ?>
 --FILE--
 <?php
-for ($i = 0; $i < 10; $i++) {
-    var_dump(bin2hex(openssl_random_pseudo_bytes($i, $strong)));
+for ($i = 1; $i < 10; $i++) {
+    var_dump(bin2hex(openssl_random_pseudo_bytes($i)));
 }
-
 ?>
 --EXPECTF--
-string(0) ""
```

Comment:
Shouldn't this be kept? I know its kind of an idiocy to test for a 0 byte random string, but hey

Reply:
This is one of the error cases that now fails closed instead of returning false. It's also the same behavior as `random_bytes()`. :)

Reply:
@SammyK Roger! Just making sure :P